docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,6 @@
 (insert scope and description here)
 
 Items:
-* [ ] Ran test & live-tests
 * [ ] Implemented (new) unit tests
```

Comment:
Next 3 can be dropped as well

Reply:
Let's retain `Implemented (new) unit tests`?
Or is there a reason we want to remove it?

Reply:
This is something already being checked by Sonar coverage

Reply:
but we didn't set it to 100% yet.
Let's retain it in the meantime.